Source/AssetRipper.Processing/AnimatorControllers/VirtualAnimationFactory.cs

```diff
@@ -39,50 +29,96 @@ public static class VirtualAnimationFactory
 		// https://github.com/ds5678/Binoculars/blob/d6702ed3a1db39b1a2788956ff195b2590c3d08b/Unity/Assets/Models/binoculars_animator.controller#L106
 		private static Utf8String BlendTreeName { get; } = new Utf8String("Blend Tree");
 
-		private static IMotion? CreateMotion(this IStateConstant stateConstant, ProcessedAssetCollection file, IAnimatorController controller, int nodeIndex)
```

Comment:
Can you reorder the methods in this class to be as close as possible to the original? The order is confusing the diff.